harness/determined/common/experimental/_session.py

```diff
@@ -0,0 +1,7 @@
+# link for backwards compatibility, since this was the original place that Session was exposed.
```

Comment:
note to self: this should not have been renamed to `_session`, that would be a breaking change.

Reply:
Actually, I will do this in a separate PR.  If I do it in this PR, it will get squished into one commit and git will decide that I rewrote this file instead of moved it elsewhere and replaced with a new one, which would break `git blame` on that file.